From 4eb8ea48c4d4036030340c9ad1b729ff24d86c98 Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Wed, 25 Mar 2020 20:56:20 +0800
Subject: [PATCH] ARM: dts: rockchip: fix aloop card index to 7 for rv11xx evb

This patch fix the aloop card index to 7, and leave the card 0
place for hw snd card.

Change-Id: I211227ad7ea495cb124b00bec9eee5039f1e1a1d
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 arch/arm/boot/dts/rv1109-evb-ddr3-v10.dts | 2 +-
 arch/arm/boot/dts/rv1109-evb-lp3-v10.dts  | 2 +-
 arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts | 2 +-
 arch/arm/boot/dts/rv1126-evb-lp3-v10.dts  | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/rv1109-evb-ddr3-v10.dts b/arch/arm/boot/dts/rv1109-evb-ddr3-v10.dts
index 8914660dea17..17a3294f3e84 100644
--- a/arch/arm/boot/dts/rv1109-evb-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1109-evb-ddr3-v10.dts
@@ -12,6 +12,6 @@
 	compatible = "rockchip,rv1109-evb-ddr3-v10", "rockchip,rv1109";
 
 	chosen {
-		bootargs = "clk_ignore_unused earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait";
+		bootargs = "clk_ignore_unused earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait snd_aloop.index=7";
 	};
 };
diff --git a/arch/arm/boot/dts/rv1109-evb-lp3-v10.dts b/arch/arm/boot/dts/rv1109-evb-lp3-v10.dts
index 15c9dce056df..fc507c9084e4 100644
--- a/arch/arm/boot/dts/rv1109-evb-lp3-v10.dts
+++ b/arch/arm/boot/dts/rv1109-evb-lp3-v10.dts
@@ -13,6 +13,6 @@
 	compatible = "rockchip,rv1109-evb-lp3-v10", "rockchip,rv1109";
 
 	chosen {
-		bootargs = "clk_ignore_unused earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait";
+		bootargs = "clk_ignore_unused earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait snd_aloop.index=7";
 	};
 };
diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
index 9b417af11b34..27efa78f860c 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10.dts
@@ -12,6 +12,6 @@
 	compatible = "rockchip,rv1126-evb-ddr3-v10", "rockchip,rv1126";
 
 	chosen {
-		bootargs = "clk_ignore_unused earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait";
+		bootargs = "clk_ignore_unused earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait snd_aloop.index=7";
 	};
 };
diff --git a/arch/arm/boot/dts/rv1126-evb-lp3-v10.dts b/arch/arm/boot/dts/rv1126-evb-lp3-v10.dts
index 52816e624c43..f9d7137e59c2 100644
--- a/arch/arm/boot/dts/rv1126-evb-lp3-v10.dts
+++ b/arch/arm/boot/dts/rv1126-evb-lp3-v10.dts
@@ -13,6 +13,6 @@
 	compatible = "rockchip,rv1126-evb-lp3-v10", "rockchip,rv1126";
 
 	chosen {
-		bootargs = "clk_ignore_unused earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait";
+		bootargs = "clk_ignore_unused earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait snd_aloop.index=7";
 	};
 };
-- 
2.35.3

